src/System.Management.Automation/engine/SessionState.cs

```diff
@@ -379,28 +379,6 @@ internal void InitializeFixedVariables()
                     RunspaceInit.PSHOMEDescription);
 
             this.GlobalScope.SetVariable(v.Name, v, false, true, this, CommandOrigin.Internal, fastPath: true);
-
-            // $Console - set the console file for this shell, if there is one, "" otherwise...
-            SetConsoleVariable();
-        }
-
-        /// <summary>
-        /// Set the $Console variable in this session state instance...
-        /// </summary>
-        internal void SetConsoleVariable()
-        {
-            // $Console - set the console file for this shell, if there is one, "" otherwise...
-            string consoleFileName = string.Empty;
-            RunspaceConfigForSingleShell rcss = ExecutionContext.RunspaceConfiguration as RunspaceConfigForSingleShell;
-            if (rcss != null && rcss.ConsoleInfo != null && !string.IsNullOrEmpty(rcss.ConsoleInfo.Filename))
-            {
-                consoleFileName = rcss.ConsoleInfo.Filename;
-            }
-            PSVariable v = new PSVariable(SpecialVariables.ConsoleFileName,
```

Comment:
`SpecialVariables.ConsoleFileName` in SpecialVariables.cs can be removed now

Reply:
will remove